tutorials/reduction_sequence_science.ipynb

```diff
@@ -294,7 +261,7 @@
     "from pykoala.corrections.flux_calibration import FluxCalibration\n",
     "from pykoala.plotting.qc_plot import qc_cube\n",
     "\n",
-    "fcal = FluxCalibration(path_to_response=os.path.join(\"products\", \"koala\", grating, 'response_HILT600_transfer_function.dat'), verbose=True)\n",
+    "fcal = FluxCalibration(path_to_response='products/response_HILT600_transfer_function.dat', verbose=True)\n",
     "\n",
     "plt.figure()\n",
```

Comment:
Same here. This basically breaks the way the products are stored in terms of the grating used.